src/traffic_layout/engine.cc

```diff
@@ -166,6 +168,15 @@ RunrootEngine::runroot_help_message(const bool runflag, const bool cleanflag, co
   return;
 }
 
+// convert a string to lower case
+static void
```

Comment:
`transform(str.begin(), str.end(), str.begin(), ::tolower);` - at that point, might as well do it inline.